components/core/src/streaming_archive/writer/File.cpp

```diff
@@ -12,7 +12,7 @@ namespace streaming_archive { namespace writer {
         if (m_is_written_out) {
             throw OperationFailed(ErrorCode_Unsupported, __FILENAME__, __LINE__);
         }
-
+        m_variable_ids = std::make_unique<std::unordered_set<variable_dictionary_id_t>>();
```

Comment:
Don't need `std::` before `std::unordered_set` (we have a `using` line at the top).